contracts/contracts/TallyNonQvFactory.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import { TallyNonQv } from "./TallyNonQv.sol";
+import { ITallySubsidyFactory } from "./interfaces/ITallySubsidyFactory.sol";
+
+/// @title TallyNonQvFactory
+/// @notice A factory contract which deploys TallyNonQv contracts.
+contract TallyNonQvFactory is ITallySubsidyFactory {
+  /// @inheritdoc ITallySubsidyFactory
+  function deploy(
+    address _verifier,
+    address _vkRegistry,
+    address _poll,
+    address _messageProcessor,
+    address _owner
+  ) public returns (address tallyAddr) {
```

Comment:
Do we need any modifiers here or anyone can deploy Tally contracts?

Reply:
anyone can deploy them, that won't cause any harm